Makefile

```diff
@@ -30,11 +30,11 @@ php = docker-compose exec -T --user $(CUID):$(CGID) php ${1}
 php-0 = docker-compose exec -T php ${1}
 
 ## Full site install from the scratch
-all: | provision back front si localize hooksymlink info
+all: | provision back front si localize content hooksymlink info
 # Install for CI deploy:review. Back & Front tasks are run in a dedicated previous step in order to leverage CI cache
-all_ci: | provision si localize hooksymlink info
+all_ci: | provision si localize content info
```

Comment:
Why removed here?

Reply:
(Unrelated to goal of this PR) Makes no sense to add git hooks using symlink in RA.